lib/deploy/activate.rb

```diff
@@ -0,0 +1,53 @@
+require 'active_support/core_ext/hash/deep_merge'
```

Comment:
We might want to extract this whole `lib/deploy/activate.rb` functionality out into identity-hostdata or some other common library given that we're duplicating it in two repos now.